tools/ncbi_blast_plus/ncbi_blastx_wrapper.xml

```diff
@@ -117,12 +123,20 @@ using the NCBI BLAST+ blastx command line tool.
 
 @FASTA_WARNING@
 
------
+-------
 
 @OUTPUT_FORMAT@
 
 -------
 
+@CLI_OPTIONS@
+
+-------
+
+@SUBSTITUTION_MATRIX_PROTEIN@
```

Comment:
You need to remove this now here and in ncbi_tblastn_wrapper.xml

Reply:
Whoops, thanks for catching that.